index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile Version</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,400;0,600;1,500&display=swap');
+        </style>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css" integrity="sha384-eoTu3+HydHRBIjnCVwsFyCpUDZHZSFKEJD0mc3ZqSBSb6YhZzRHeiomAUWCstIWo" crossorigin="anonymous">
+</head>
+<body>
+    <main class="container">
+        <div class="work1">
+            <img src="Snapshoot.png" alt="">
+           
+            <h2>Tonic</h2>
+            <div id="frame">
+                <div id="text1"><h5>CANOPY</h5></div>
```

Comment:
hey in the required changes, you have said that I should use the Figma that I used for the toolbar and headline from my previous milestone. I don't understand because it is the same template that I have used. Any further details would really help. Thank

Reply:
@gtekle 
hey in the required changes, you have said that I should use the Figma that I used for the toolbar and headline from my previous milestone. I don't understand because it is the same template that I have used. Any further details would really help. Thank you